### ISSUE TITLE
want the details damage meter to be adaptive for the Mop 5.4.8

### ISSUE THREAD

**Issue by carbonink** [Bunny67/Details-WotLK#68]
Hi, Author, I see that you have already edited the  Details interface of the retail wow version so that  it is adative for the wlk 3.3.5. it is great! And  I beg you for coding the Details interface so that  it is adaptive for the Mop 5.4.8 Version. As there is a server named Tauriwow  and the Mop(mists of padanria) is avaliable. Thank you again!

**Comment by andrew6180** [comment]
https://www.curseforge.com/wow/addons/details/files/all?filter-game-version=2020709689%3A474

**Comment by carbonink** [comment]
No, I think you must misunderstand me.  The old version of  the Details Interface  released on Oct 12, 2014 is lack of many useful functions and features compared with the retail version of the Details interface supported for the Wow game version 9.1.  And the Details-WotLK came as the same reason.  As many useful and important improvements are added to the latest retail wow details interface,  so many authors edit it so that  it is adaptive for classic, tbc, wotlk and so on without discarding the new and useful funtions and features.  So i request you author could help the players who love playing the Mop version to make out the New Details damage meter interface instead of the Old useless version released in 2014.  It must be a hard work. So thank you for your efforts! 
（P.S.  I offer the link of the most popular Mist of Padaria(Mop) Version of Wow avaliable for Soo of 5.4.8 below to help you to test the edited version of Details interface if it really works well, Thank you again.
https://www.tauriwow.com）

**Comment by Lemonadez** [comment]
I think its better for 4.3.4 Cata first. i think that one is easier than mop..since wotlk-cata almost identical in api.
